webapp/Connector/src/plugin/Messaging.js

```diff
@@ -118,6 +118,7 @@ Ext.define('Connector.plugin.Messaging', {
                 x: this.calculateX(this, box, msg),
                 y: this.calculateY(this, box, msg),
                 listeners: listeners,
+                width: 150,
```

Comment:
This fixes the button overflow issue when clearing the active filter. In other places we typically provide a width when using an Ext.Window component and this seems to make the element size more accurately on chrome and safari.